tests/data/bundle.yaml

```diff
@@ -0,0 +1,97 @@
+description: A minimal two-machine Kubernetes cluster, appropriate for development.
+series: focal
+machines:
+  '0':
+    constraints: cores=2 mem=4G root-disk=16G
+    series: focal
+  '1':
+    constraints: cores=4 mem=4G root-disk=16G
+    series: focal
+services:
+  containerd:
+    charm: cs:~containers/containerd
+    channel: edge
+    resources: {}
+  easyrsa:
+    charm: cs:~containers/easyrsa
+    channel: edge
+    num_units: 1
+    resources:
+      easyrsa: 5
```

Comment:
I would remove resource revisions from the bundle. No point specifying them when we don't specify charm revisions.